ProteoformSuiteInternal/Component.cs

```diff
@@ -35,6 +35,7 @@ public class Component
         public bool calculating_properties { get; set; } = false;
         private int num_detected_intervals { get; set; }
         public bool accepted { get; set; }
+        public int charge_states_count { get; set; }
```

Comment:
I don't thnik the changes in Component.cs are necessary... I would model the component reader changes off of the .tsv code that already exists and use the constructor that already existed